src/sentry/api/endpoints/organization_incident_index.py

```diff
@@ -67,15 +67,15 @@ def get(self, request, organization):
         )
 
         query_status = request.GET.get("status")
-
-        if query_status == "critical":
-            incidents = incidents.filter(status=IncidentStatus.CRITICAL.value)
-        elif query_status == "warning":
-            incidents = incidents.filter(status=IncidentStatus.WARNING.value)
-        elif query_status == "closed":
-            incidents = incidents.filter(status=IncidentStatus.CLOSED.value)
-        elif query_status == "open":
-            incidents = incidents.exclude(status=IncidentStatus.CLOSED.value)
+        if query_status is not None:
+            if query_status == "open" or query_status == "active":
```

Comment:
Do we still need to support `open`?

Reply:
I was actually thinking of removing "active". They are currently equivalent and I think we should just choose one.

Thinking "open" because it more intuitively feels like "not closed", which makes it feel simpler - and because we have been using it historically.

Thoughts?
Thinking the only place we'd see "active" is on the ui.